<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify</class>
 <widget class="QMainWindow" name="modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6">
     <widget class="QPushButton" name="modiftyLogout_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #D7BDE2;
    color: black;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #AF7AC5;
}
QPushButton:pressed {
    background-color: #884EA0;
}</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maths Marks -</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student Marks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>New Marks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Chemistry Marks -</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3" colspan="2">
     <widget class="QPushButton" name="makeChanges">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #5DADE2;
    color: black;
}

QPushButton:hover {
    background-color: #3498DB;
}

QPushButton:pressed {
    background-color: #2E86C1;
    color: white;
}
</string>
      </property>
      <property name="text">
       <string>Make Changes</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>31</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="idModifySearch">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #A8DADC;
    color: #1D3557;
}

QPushButton:hover {
    background-color: #89C2D9;
}

QPushButton:pressed {
    background-color: #457B9D;
    color: #F1FAEE;
}
</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="studentidmodify">
      <property name="placeholderText">
       <string>Enter Student ID Here</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QDoubleSpinBox" name="mathsNew">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QDoubleSpinBox" name="physicsNew">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="modifyHome_btn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #85C1E9;
    color: black;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #5DADE2;
}
QPushButton:pressed {
    background-color: #3498DB;
}</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QDoubleSpinBox" name="chemNew">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Physics Marks - </string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Previous Marks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Student ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_8">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
